docs/logo/r-package-hex-logo.svg

```diff
@@ -0,0 +1,31 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="819" height="945" viewBox="0 0 819 945">
+  <metadata><?xpacket begin="﻿" id="W5M0MpCehiHzreSzNTczkc9d"?>
+<x:xmpmeta xmlns:x="adobe:ns:meta/" x:xmptk="Adobe XMP Core 5.6-c142 79.160924, 2017/07/13-01:06:39        ">
+   <rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+      <rdf:Description rdf:about=""/>
+   </rdf:RDF>
+</x:xmpmeta>
```

Comment:
@guolinke I see now that this svg just has the PNG data base64-encoded. Is it possible to provide a "true" SVG like this?

https://github.com/microsoft/LightGBM/blob/master/docs/logo/LightGBM_logo_black_text.svg?short_path=a341128

I think that would be more friendly to edit in the future if we need to.

Reply:
sorry, I have very little experience editing image files  😂

Reply:
I can do this.

Reply:
thanks!

Reply:
Please try this file. I "fixed" source file as well: made red triangle vector instead of bitmap (it was the reason of embeded image in SVG) and increased size of the page. Now online convertors should work OK with this file, I believe.

Reply:
GitHub doesn't want to allow me to upload those files 😢 

May I create a PR in your repo with files?

![image](https://user-images.githubusercontent.com/25141164/91494759-04336d00-e8c2-11ea-808f-f810220a4ab8.png)

Reply:
OK, it was not a temporary issue. I cannot upload files with the same error right now.

I forgot that this PR is from main repo's branch, so I just opened #3340.

Reply:
oh I've seen that problem! I remember having an issue uploading a `.html` file as well. Thanks for the PR, I've merged it here.